feg/gateway/services/aaa/radius/dae/dae_external.go

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright (c) Facebook, Inc. and its affiliates.
+// All rights reserved.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+// +build !with_builtin_radius
+
+// package dae implements Radius Dynamic Authorization Extensions API (https://tools.ietf.org/html/rfc5176)
+package dae
+
+import (
+	"context"
+
+	"magma/feg/cloud/go/protos/mconfig"
+	"magma/feg/gateway/registry"
+	"magma/feg/gateway/services/aaa/protos"
+)
+
+type extDAEServer struct{}
+
+// NewDAEServicer returns default servicer using external, registry based service
+func NewDAEServicer(*mconfig.RadiusConfig) DAE {
```

Comment:
why do you need RadiusConfig if you are not using it on the creation

Reply:
to keep API identical between built in & external